kotlinx-coroutines-core/common/src/channels/BufferedChannel.kt

```diff
@@ -248,7 +248,7 @@ internal open class BufferedChannel<E>(
      * Note that this implementation always invokes [suspendCancellableCoroutineReusable],
      * as we do not care about broadcasts performance -- they are already deprecated.
```

Comment:
```suggestion
```

Reply:
Also, what did this comment mean? `suspendCancellableCoroutineReusable` is a *slower* internal version of `suspendCancellableCoroutine` that we use internally as a performance de-optimization, and now that `BroadcastChannel` is deprecated, we're free to make it fast, right?

Reply:
I believe this is just a miswording, asked Nikita to verify just in case.

Reply:
The idea was "it doesn't have non-suspending fast-path"